### PR TITLE
fix: fix cream supply approval error and borrow limit display error

### PR DESCRIPTION
**Pull request by x48-crypto**

### CODE REVIEW

File: src/stores/store.jsx [hunk 1]
```diff
@@ -2038,10 +2039,7 @@ class Store {
         ethAllowance = (allowance*10**asset.decimals).toFixed(0);
       }
 
-      var amountToSend = web3.utils.toWei('999999999', "ether")
-      if (asset.decimals !== 18) {
-        amountToSend = (999999999*10**asset.decimals).toFixed(0);
-      }
+      const amountToSend = MAX_UINT256;
```
Comment (x48-crypto) [inline src/stores/store.jsx:2042]: Hard code to MAX_UINT256

File: src/components/lending/borrowAsset.jsx [hunk 2]
```diff
@@ -269,6 +269,9 @@ class BorrowAsset extends Component {
     } else {
       theLimitUsed = (limitUsed)*100/limit
     }
+    if (isNaN(theLimitUsed)) {
+      theLimitUsed = 0
+    }
```
Comment (x48-crypto) [inline src/components/lending/borrowAsset.jsx:274]: Fix "NaN%" display error

File: src/components/lending/supplyAsset.jsx [hunk 3]
```diff
@@ -293,10 +293,10 @@ class SupplyAsset extends Component {
       loading,
     } = this.state
 
-    let theLimitUsed = (limitUsed)*100/limit
+    let theLimitUsed = (limitUsed)*100/limit || 0
 
     if(asset.collateralEnabled) {
-      if(supplyAmount && supplyAmount !== '') {
+      if(supplyAmount && supplyAmount !== '' && limit !== 0) {
```
Comment (x48-crypto) [inline src/components/lending/supplyAsset.jsx:299]: fix "-Infinity%" display error

**Review by antonnell — APPROVED** [review]
All looks good.

**Pull request merged**